### PR TITLE
Skip fp16 accuracy test for group_norm and cumulative_trapezoid

### PR DESCRIPTION
**Pull request by ggggxm**
- cumulative_trapezoid底层调用cumsum API，在fp16精度下，paddle计算正确，torch计算错误
- einsum 添加paddle error dismiss，因为matmul不支持N的维度大于1073741823
- group_norm在fp16反向情况下，如果结果绝对值大，绝对误差较大，但相对误差较小；如果结果绝对值小，绝对误差小，相对误差较大，认为可以忽略。
截屏2025-07-25 15 02 50

截屏2025-07-25 15 05 02

### CODE REVIEW

File: tester/api_config/torch_error_skip.txt [hunk 1]
```diff
@@ -14,6 +14,15 @@ paddle.logcumsumexp(Tensor([10, 429496730],"float16"), dtype="float16", axis=Non
...
+paddle.nn.functional.group_norm(Tensor([357913942, 4, 3],"float16"), num_groups=2, weight=Tensor([4],"float16"), bias=Tensor([4],"float16"), data_format="NCL", )
+paddle.nn.functional.group_norm(Tensor([357913942, 3, 4],"float16"), num_groups=2, weight=Tensor([4],"float16"), bias=Tensor([4],"float16"), data_format="NLC", )
+paddle.nn.functional.group_norm(Tensor([2, 536870913, 4],"float16"), num_groups=2, weight=Tensor([4],"float16"), bias=Tensor([4],"float16"), data_format="NLC", )
+paddle.nn.functional.group_norm(Tensor([2, 3, 89478486, 2, 4],"float16"), num_groups=2, weight=Tensor([4],"float16"), bias=Tensor([4],"float16"), data_format="NDHWC", )
+paddle.nn.functional.group_norm(Tensor([2, 3, 2, 89478486, 4],"float16"), num_groups=2, weight=Tensor([4],"float16"), bias=Tensor([4],"float16"), data_format="NDHWC", )
+paddle.nn.functional.group_norm(Tensor([2, 268435457, 2, 4],"float16"), num_groups=2, weight=Tensor([4],"float16"), bias=Tensor([4],"float16"), data_format="NHWC", )
+paddle.nn.functional.group_norm(Tensor([2, 134217729, 2, 2, 4],"float16"), num_groups=2, weight=Tensor([4],"float16"), bias=Tensor([4],"float16"), data_format="NDHWC", )
```
Comment (cangtianhuang) [inline tester/api_config/torch_error_skip.txt:25]: 这个不能通过调整 atol 和 rtol 放行吗？

File: tester/base_config.yaml [hunk 2]
```diff
@@ -19,6 +19,7 @@ paddle_error_dismiss:
   paddle.Tensor.inverse: "(Unimplemented) cublasMatInv does not support batch_size > 65536."
   paddle.incubate.nn.functional.fused_rms_norm : "(Fatal) Failed to launch RMSNorm kernel"
   paddle.nn.functional.ctc_loss: "(InvalidArgument) The total number of elements "
+  paddle.nn.functional.group_norm: "(Unimplemented) GroupNorm kernel launch failed"
```
Comment (cangtianhuang) [inline tester/base_config.yaml:22]: 这个报错是什么原因呀？

**Review by cangtianhuang — CHANGES_REQUESTED** [review]

**Review by cangtianhuang — APPROVED** [review]
LGTM
